src/typ/expr.rs

```diff
@@ -384,6 +393,7 @@ impl<'a, 'b, 'c> ExprTyper<'a, 'b, 'c> {
         })
     }
 
+    #[allow(clippy::map_unwrap_or)]
```

Comment:
What does this mean?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#map_unwrap_or It worked in most cases, but was a false positive on this particular function.

Reply:
If it is giving false positives let's disable this globally as it's not ready to be used.